xAPI.md

```diff
@@ -237,43 +242,38 @@ JSON allows for objects to have properties that contain empty objects. This is n
 
 <a name="def-activity" />
 
-__Activity__: An Activity is a type of Object making up the “this” in I did “this”; it is something 
+__Activity__: An Activity is a type of Object making up the “this” in “I did this”; it is something 
 with which an Actor interacted. It can be a unit of instruction, experience, or performance that is 
 to be tracked in meaningful combination with a Verb. Interpretation of Activity is broad, meaning that 
 Activities can even be tangible objects such as a chair (real or virtual). In the statement "Anna 
 tried a cake recipe", the recipe constitutes the Activity in terms of the xAPI statement. Other 
 examples of activities include a book, an e-learning course, a hike or a meeting.
 
-<a name="def-activity-provider" />
-
-__Activity Provider (AP)__: The software object that is communicating with 
```

Comment:
I think you know I'm against removing this term, but I recognize the group disagrees so won't go over that. 

As it is being removed, we'll need to take action to manage the confusion this will cause. Leaving the definition in with a reference to the fact in changed (as we do with 'Tin Can') will help a little.